Jenkinsfile.cloud

```diff
@@ -69,4 +73,26 @@ node(env.NODE) {
             """
         }
     }
+
+    stage("Create AMI") {
+        docker.image(DOCKER_IMG).inside(DOCKER_ARGS) {
+            // not packaged yet, just build from src for now
+            sh "dnf install -y git golang"
+            sh "git clone https://github.com/coreos/mantle"
+            sh "mantle/build ore"
+
+            withCredentials([[$class: 'AmazonWebServicesCredentialsBinding',
+                              credentialsId: env['AWS_CREDENTIALS']]]) {
```

Comment:
So, does this Credentials link to a AWS EC2 VM? Where does it get consumed? Is jenkins able to link the credential ID to a EC2 VM or...? /me asking with almost no context tho :p.

Reply:
@peterbaouoft Jenkins provides said ``AWS_CREDENTIALS`` environment variable. The value is stored in jenkins itself. Using it allows further AWS commands to be used with said credentials.

Reply:
Ah, I think I get it now. I was unsure what s3 was b4, that is why I kinda have the confusion :p. It seems like the credentials was used for access for s3(hopefully). Thanks for the clarification =).